notebooks/amazon_forest_notebook.py

```diff
@@ -208,32 +207,52 @@
 
 # <codecell>
 
-from keras.callbacks import ModelCheckpoint
+from tensorflow.contrib.keras.api.keras.callbacks import ModelCheckpoint
 
 filepath="weights.best.hdf5"
-checkpoint = ModelCheckpoint(filepath, monitor='val_acc', verbose=1, save_best_only=True, mode='max')
+checkpoint = ModelCheckpoint(filepath, monitor='val_acc', verbose=1, save_best_only=True)
 
 # <markdowncell>
 
-# ## Create the neural network definition
+# ## Define and Train model
+
+# <markdowncell>
+
+# Here we define the model and begin training. 
+# 
+# Before starting the training process, you should first set a learning rate annealing optimization schedule by choosing a series of learning rates (learn_rates) with corresponding number of epochs for each (epochs_arr).
+# 
+# Alternatively, if you just want to run one training session at a fixed learning rate and num epochs you can just input one entry for each of these. 
 
 # <codecell>
 
 classifier = AmazonKerasClassifier()
 classifier.add_conv_layer(img_resize)
 classifier.add_flatten_layer()
 classifier.add_ann_layer(len(y_map))
-train_losses, val_losses, fbeta_score = classifier.train_model(x_train, y_train, epochs, batch_size, validation_split_size=validation_split_size)
+
+train_losses, val_losses = [], []
+epochs_arr = [20, 5, 5]
+learn_rates = [0.001, 0.0001, 0.00001]
+for learn_rate, epochs in zip(learn_rates, epochs_arr):
+    tmp_train_losses, tmp_val_losses, fbeta_score = classifier.train_model(x_train, y_train, learn_rate, epochs, 
+                                                                           batch_size, validation_split_size=validation_split_size, 
+                                                                           train_callbacks=[checkpoint])
+    train_losses.append(tmp_train_losses)
+    val_losses.append(tmp_val_losses)
```

Comment:
This will append lists into lists, instead we want to append the value to the end of the original list so the correct behaviour here would be using `+=` instead of `append`